app/src/main/java/com/androidavanzado/covid19stadistics/service/ApiCovidServicePresentationImpl.kt

```diff
@@ -0,0 +1,15 @@
+package com.androidavanzado.covid19stadistics.service
+
+import au.com.carsales.basemodule.context
+import com.example.apicovidmodule.model.StadisticsApi
+import com.example.apicovidmodule.storage.Result
+import com.example.routercovidservices.getApiCovidService
+import com.example.routercovidservices.presentation.ApiCovidServicePresentation
+import java.util.*
+
+open class ApiCovidServicePresentationImpl : ApiCovidServicePresentation {
+
+    override suspend fun getData(date: String): Result<StadisticsApi> {
+        return context?.getApiCovidService()?.getDataFromApi(date)!!
```

Comment:
💡 The `context?.getApiCovid Service()` should be passed by parameters because this layer shouldn't have UI dependencies in it to be more pure and more testable